addons/website/views/snippets/s_cta_box.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_box" name="CTA Box">
+    <section class="s_cta_box pt80 pb80">
```

Comment:
Could you try `data-name="Section"` here?
I'm not sure if the editor allows that since it's the first tag in the snippet.

Rationale:
The problem I see is that "CTA Box" gets too much attention and it's not very self explicative.
When looking at the sidebar, I have the feeling that's the most important element there, while its role it's just to provide support for the card that, instead, uses the very generic name "Block".

| |
|--------|
| ![image](https://github.com/user-attachments/assets/e5970ccf-61da-4911-8f45-62525e0d7251) |

Reply:
Renaming it to Section implies to use "Section" within the string of the snippet in `snippets.xml`, this may be a bit overkilled. Instead, I opted for "Box Call to Action" for the snippet name and "Card" for the card element.

What do you think?

Reply:
LGTM 👍